src/index.html

```diff
@@ -56,40 +56,40 @@ <h3>Your Current Team:</h3>
         <h3>Yesterdays Journal:</h3>
         <div class="journal dashboard-journal">
           <h6>Rating:</h6>
-          <img src="./src/svgs/Star.svg" alt="star icon" class="star-gap" />
-          <img src="./src/svgs/Star.svg" alt="star icon" class="star-gap" />
-          <img src="./src/svgs/Star.svg" alt="star icon" class="star-gap" />
+          <img src="./svgs/Star.svg" alt="star icon" class="star-gap" />
+          <img src="./svgs/Star.svg" alt="star icon" class="star-gap" />
+          <img src="./svgs/Star.svg" alt="star icon" class="star-gap" />
           <img
-            src="./src/svgs/Star.svg"
+            src="./svgs/Star.svg"
             alt="star icon"
             class="star-grey star-gap"
           />
           <img
-            src="./src/svgs/Star.svg"
+            src="./svgs/Star.svg"
             alt="star icon"
             class="star-grey star-gap"
           />
 
           <h6>Comprehension:</h6>
-          <img src="./src/svgs/Rectangle.svg" alt="rectangle icon" />
-          <img src="./src/svgs/Rectangle.svg" alt="rectangle icon" />
-          <img src="./src/svgs/Rectangle.svg" alt="rectangle icon" />
-          <img src="./src/svgs/Rectangle.svg" alt="rectangle icon" />
-          <img src="./src/svgs/Rectangle.svg" alt="rectangle icon" />
-          <img src="./src/svgs/Rectangle.svg" alt="rectangle icon" />
-          <img src="./src/svgs/Rectangle.svg" alt="rectangle icon" />
+          <img src="./svgs/Rectangle.svg" alt="rectangle icon" />
+          <img src="./svgs/Rectangle.svg" alt="rectangle icon" />
+          <img src="./svgs/Rectangle.svg" alt="rectangle icon" />
+          <img src="./svgs/Rectangle.svg" alt="rectangle icon" />
+          <img src="./svgs/Rectangle.svg" alt="rectangle icon" />
+          <img src="./svgs/Rectangle.svg" alt="rectangle icon" />
+          <img src="./svgs/Rectangle.svg" alt="rectangle icon" />
           <img
-            src="./src/svgs/Rectangle.svg"
+            src="./svgs/Rectangle.svg"
             alt="rectangle icon"
             class="icon-grey"
           />
           <img
-            src="./src/svgs/Rectangle.svg"
+            src="./svgs/Rectangle.svg"
             alt="rectangle icon"
             class="icon-grey"
           />
           <img
-            src="./src/svgs/Rectangle.svg"
+            src="./svgs/Rectangle.svg"
             alt="rectangle icon"
             class="icon-grey"
           />
```

Comment:
These could all be grouped in an element, just to structure the code a bit. If the size of these assets wouldn't be so restricted you'd probably need to use a layout here anyway